app/models/comment.rb

```diff
@@ -0,0 +1,8 @@
+class Comment < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  def update_counter
+    post.update(comments_counter: post.comments.count)
+  end
+end
```

Comment:
**OPTIONAL**

- [ ] You can use one of the [Rails Active Callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) - after_save to call the #update_counter method every time a new comment has been saved to the database, then make the update_counter method private.

```ruby
  after_save :update_counter

  private

  def update_counter
    post.update(comments_counter: post.comments.count)
  end
```

> Please consider this very cool helper in your other classes.